spring-cloud-stream-binder-rabbit-docs/src/main/asciidoc/overview.adoc

```diff
@@ -98,6 +98,11 @@ spring.cloud.stream.rabbit.binder.compressionLevel::
 See `java.util.zip.Deflater`.
 +
 Default: `1` (BEST_LEVEL).
+spring.cloud.stream.binder.connection-name-prefix::
+  A connection name prefix used to name the connection(s) created by this binder.
+The name will be this prefix followed by `#n`, where n increments each time a new connection is opened.
++
+Defauklt: none (Spring AMQP default).
```

Comment:
Typo: default

Also I wonder if this descriptions in the Docs can be also generated from the mention above JavaDocs.
The closest way we do it in the App Starters with the Docs plugin.
Otherwise, when we fix those JavaDocs, we may end up with an inconsistency having here different description.

Right, different story, but I would like to raise it as it follows.

Reply:
I don't think we can generate these docs - for example see how complex they are for `adminAddresses` and `nodes`.